package.json

```diff
@@ -69,6 +69,8 @@
 		"@oslojs/crypto": "^1.0.1",
 		"@oslojs/encoding": "^1.1.0",
 		"drizzle-orm": "^0.39.3",
+		"form-data": "^4.0.1",
```

Comment:
I think this should be built-in to Node.js by default now and it shouldn't need any extra dependencies for form-data. Did you run into some issues with Node implementation?